lib/kdump_utils.pm

```diff
@@ -96,6 +96,12 @@ sub activate_kdump {
     my @tags = qw(yast2-kdump-disabled yast2-kdump-enabled yast2-kdump-restart-info yast2-missing_package yast2_console-finished);
     do {
         assert_screen \@tags, 300;
+        # for ppc64le we need increase kdump memory, see bsc#986196 and bsc#1070397
```

Comment:
boths bugs do not seem to explain the problem. bsc#986196 references 32GB of RAM and bsc#1070397 is VERIFIED WORKSFORME. I am sure there are more bugs open about that one reported by us and referencing openQA tests. Also, I would use record_soft_failure to apply the below workaround.

However, what I still consider missing is first the *detection* of the actual issue before applying a workaround.

Reply:
What do you mean with "please give it a try"? How about you trigger tests on production using http://open.qa/docs/#_triggering_tests_based_on_an_any_remote_git_refspec_or_open_github_pull_request ?